pkg/collector/corechecks/servicediscovery/module/envs_test.go

```diff
@@ -103,3 +105,67 @@ func memfile(name string, b []byte) (int, error) {
 
 	return fd, nil
 }
+
+func TestEnvScanner(t *testing.T) {
+	curPid := os.Getpid()
+	proc, err := process.NewProcess(int32(curPid))
+	require.NoError(t, err)
+
+	envMapGood, err := getEnvs(proc)
+	require.NoError(t, err)
+
+	// extract environment variables using scanner
+	envMapCurr, err := getEnvsUsingScan(proc, false)
```

Comment:
Why implement and test the false case if it's not used anywhere?

Reply:
I removed second parameter in this function